chain/find.go

```diff
@@ -222,44 +222,42 @@ func (g *GapIndexer) findTaskEpochGaps(ctx context.Context) (visor.GapReportList
 		ctx,
 		&result,
 		`
-select height, task
-	from (
-
-	-- incomplete_heights_and_their_completed_tasks: for every incomplete
-	-- heightin incomplete_task_counts we will take all processing_reports
-	-- with the same height and return height,tasks which are 'OK'
-	-- which provides all incomplete heights and their completed tasks
-	select distinct vpr.task, vpr.height
-	from visor_processing_reports vpr
-	right join(
-
-		-- incomplete_task_counts: count of all tasks by height,status
-		-- and filter for status = 'OK' and all task counts which
-		-- don't equal len(AllTasks)
-		select incomplete_task_counts.height, status, count(incomplete_task_counts.height) cheight
-		from (
-
-			-- all unique height,tasks,status within range of interest
-			select task, height, status
-			from visor_processing_reports
-			where height >= ?2 and height <= ?3
-			group by height, task, status
-			order by height desc, task
-
-		) incomplete_task_counts
-		where status = ?4
-		group by incomplete_task_counts.height, incomplete_task_counts.status
-		having count(incomplete_task_counts.height) != ?0
-
-	) incomplete_heights
+with
+
+-- ranked_tasks: filters by height and prepares ranking status of tasks by int
+-- only considers heights with incomplete task count
+ranked_tasks as (
```

Comment:
I would like names to be descriptive of their domain-specific properties rather than functional properties. This will make it easier to reason about the purpose as well as the mechanics in complex queries like this instead of just the latter.

Ex: `ranked_tasks` -> `status_weighted_tasks_from_incomplete_epochs`

Aside: I have no problem with (and would prefer) verbosity in my names if they add value over terse names. Particularly if they are separated by many lines. But please use your judgement on names. #namingishard

Reply:
@placer14 thanks! I'll try to be more descriptive  (and more correct) with names.